### PR TITLE
ci: add mongodb connection test to .circleci\config.yml and fix some minor errors in README

### PR DESCRIPTION
**Pull request by yifeili98**
Add MongoDB connection test to CI.
CI won't pass if the connection to FHDA cluster cannot be established.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,9 +50,20 @@ jobs:
...
+    steps:
+      - checkout
+      - run:
+          command:
+            "mongo mongodb+srv://fhdatimedb-jjsjm.mongodb.net/test --eval 'db.serverStatus()'"
+            
+            
 workflows:
   version: 2
   build_and_test:
```
Comment (hk-mp5a3) [inline .circleci/config.yml:65]: "build_and_test"但是里面似乎只包含了test? 是为了方便后续需要再加入“build”部分吗？

**Review by hk-mp5a3 — COMMENTED** [review]
nit(跟这个commit不相关): line 44-45是不是可以放成一行?

File: .circleci/config.yml (hunk 1 above)
Comment (yifeili98) [reply .circleci/config.yml:65]: Maybe? Need to confirm with William

**Comment by yifeili98** [comment]
nit: forgot to put newline at EOF, will fix it in next commit

File: .circleci/config.yml [hunk 2]
```diff
@@ -50,9 +50,21 @@ jobs:
...
           command: |
             /bin/sh scripts/commitlint_range.sh
+  test_mongodb:
+    docker:
+      - image: circleci/mongo:latest
+    steps:
+      - checkout
+      - run:
+          command:
+            "mongo mongodb+srv://fhdatimedb-jjsjm.mongodb.net/test --eval 'db.serverStatus()'"
```
Comment (kis87988) [inline .circleci/config.yml]: I am thinking about we should put this link in the env to aovid security issue.

Comment (yifeili98) [reply .circleci/config.yml]: should we use an environment variable?

Comment (kis87988) [reply .circleci/config.yml]: should be

**Review by kis87988 — APPROVED** [review]

**Review by biankaiwen111 — APPROVED** [review]
LGTM

**Pull request merged**